src/oop/app/src/Transporters/GuzzleAdapter.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace src\oop\app\src\Transporters;
+
+use GuzzleHttp\Client;
+
+class GuzzleAdapter implements TransportInterface
+{
+    private Client $client;
+
+    public function __construct(){
+        $this->client = new Client();
```

Comment:
please do not use new Client, inject it as dependency

Reply:
Done in last commit.